plasmapy/classes/tests/test_Plasma.py

```diff
@@ -97,6 +97,84 @@ def test_Plasma3D_derived_vars():
     assert np.allclose(test_plasma.alfven_speed.value, 10.92548431)
 
 
+class Test_PlasmaBlobRegimes():
+    def test_intermediate_and_both_energies(self):
```

Comment:
Add docstrings for all of these tests to be clear on what is being tested. The names themselves are not so clear... what does "intermediate" and "both energies" refer to? This is important because eventually `PlasmaBlob` will have all sorts of parameters and not just coupling parameter and Fermi energy.